js/adapt-assessmentArticleModel.js

```diff
@@ -84,15 +84,16 @@ define([
 
         _setAssessmentOwnershipOnChildrenModels: function() {
             //mark all children components as belonging to an assessment
+            var assessmentConfig = this.get('_assessment');
+            var blockModelConfig = {
```

Comment:
Let's call this something like `childConfig` since it lives on both blocks and components?